README.md

```diff
@@ -17,43 +17,65 @@ You can use a stack image to do any of the following (and more):
+
+```bash
+docker run -p 8888:8888 jupyter/scipy-notebook:b418b67c225b
+```
+
 It then starts a container running a Jupyter Server and exposes the server on host port 8888.
 The server logs appear in the terminal.
+
 Visiting `http://<hostname>:8888/?token=<token>` in a browser loads JupyterLab,
-where `hostname` is the name of the computer running docker and `token` is the secret token printed in the console.
+where:
+
+- `hostname` is the name of the computer running docker
+- `token` is the secret token printed in the console
+
 The container remains intact for restart after the Jupyter Server exits.
 
-```bash
-docker run -p 8888:8888 jupyter/scipy-notebook:b418b67c225b
-```
+**Example 2:**
 
-**Example 2:** This command performs the same operations as **Example 1**, but it exposes the server on host port 10000 instead of port 8888.
-Visiting `http://<hostname>:10000/?token=<token>` in a browser loads JupyterLab,
-where `hostname` is the name of the computer running docker and `token` is the secret token printed in the console.
+This command performs the same operations as **Example 1**, but it exposes the server on host port 10000 instead of port 8888.
 
 ```bash
 docker run -p 10000:8888 jupyter/scipy-notebook:b418b67c225b
 ```
 
-**Example 3:** This command pulls the `jupyter/datascience-notebook` image tagged `b418b67c225b` from Docker Hub if it is not already present on the local host.
-It then starts an _ephemeral_ container running a Jupyter Server and exposes the server on host port 10000.
-The command mounts the current working directory on the host as `/home/jovyan/work` in the container.
-The server logs appear in the terminal.
 Visiting `http://<hostname>:10000/?token=<token>` in a browser loads JupyterLab,
-where `hostname` is the name of the computer running docker and `token` is the secret token printed in the console.
-Docker destroys the container after Jupyter Server exit, but any files written to `~/work` in the container remain intact on the host.
+where:
+
+- `hostname` is the name of the computer running docker
+- `token` is the secret token printed in the console.
+
+**Example 3:**
+
+This command pulls the `jupyter/datascience-notebook` image tagged `b418b67c225b` from Docker Hub if it is not already present on the local host.
+It then starts an _ephemeral_ container running a Jupyter Server and exposes the server on host port 10000.
 
 ```bash
 docker run --rm -p 10000:8888 -v "${PWD}":/home/jovyan/work jupyter/datascience-notebook:b418b67c225b
 ```
 
+The use of the `-v` flag in the command mounts the current working directory on the host (`S{PWD}` in the example command) as `/home/jovyan/work` in the container.
+The server logs appear in the terminal.
+
+Visiting `http://<hostname>:10000/?token=<token>` in a browser loads JupyterLab,
+where:
+
+- `hostname` is the name of the computer running docker
+- `token` is the secret token printed in the console.
```

Comment:
I don't think this duplication is needed, because it has already been explained above.